ykcapi/src/lib.rs

```diff
@@ -156,8 +156,10 @@ pub extern "C" fn __ykrt_reconstruct_frames(newframesptr: *const c_void) {
     unsafe {
         asm!(
             // The first 8 bytes of the new frames is the size of the map, needed for copying it
-            // over. Move it into rdx.
+            // over. Move it into rdx, but reduce it by 8 bytes, since we'll move the pointer along
+            // next.
```

Comment:
I'm not sure what you mean by "move the pointer along". Let's be more precise.

Reply:
[03eb360](https://github.com/ykjit/yk/pull/616/commits/03eb3608dc24d9ab3ca3909900a3f1e188c9aae6)